website/docs/configuration/modules.html.md

```diff
@@ -361,25 +361,23 @@ setting inside a `terraform` block.
 
 ## Multiple Instances of a Module
 
-A particular module source can be instantiated multiple times:
+Use the `count` or `for_each` arguments to create multiple instances of a module.
```

Comment:
I like that "multiple instances" is now being applied to something that the terminology is more appropriate for! :smile_cat: 

Do you think it would be worth retaining a note somewhere else on the page that it's possible to call the same module multiple times with separate blocks? That was what the old version of this section was trying to draw attention to, and I think it'll still be the right answer for some situations where the calls are not similar enough to be convenient to express with the repetition features.

The initial idea I have for this would be to open this section by _very briefly_ talking about the fact that the same module can be called multiple times with multiple `module` blocks, and then transition into describing `count` and `for_each` as a way to do that repetition programmatically and dynamically.

Reply:
I do think there's a place for that, but that could be in a next step -- another docs task is adding information about the limitations when you leverage module expansion, so that's the section where it will be natural to talk about having separate explicit blocks.